src/nodes/ImageView.vue

```diff
@@ -260,12 +260,15 @@ export default {
 
 	.image__view {
 		text-align: center;
-
-		.image__main {
-			max-height: 40vh;
+		img {
+			width: 100%;
```

Comment:
```suggestion
			max-width: 100%;
```

Should still be max-width, otherwise portrait images would be stretched.